framework/archiver/listeners.py

```diff
@@ -0,0 +1,41 @@
+from framework.tasks.handlers import enqueue_task
+from framework.archiver.tasks import archive, send_success_message
+from framework.archiver.utils import (
+    link_archive_provider,
+)
+from framework.archiver import (
+    ARCHIVER_SUCCESS,
+    ARCHIVER_FAILURE,
+)
+from framework.archiver.exceptions import ArchiverCopyError
+
+from website.project import signals as project_signals
```

Comment:
If you have to import anything website I would discourage putting anything in framework.
This is directly related to project, why is it not in `website.project`?